spec/services/code_climate/code_climate_spec_helper.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+module CodeClimateSpecHelper
```

Comment:
if this is a mock, it should be inside `spec/support/helpers/` and be called something like `code_climate_api_mocker`

Reply:
Done it. Thanks!